packages/create-graphback/src/init/starterTemplates.ts

```diff
@@ -113,6 +114,9 @@ export const allTemplates: Template[] = [
   }
 ];
 
+const externalTemplatesArray: Template[] = externalTemplates;
```

Comment:
I would also improve how we present those templates - Graphback and community templates when executing create-graphback cmd

Reply:
Thanks for reviewing the code @wtrocki. Do you have any suggestions about how I could go on implementing this? All templates must still reside in the `allTemplates` array for the code to function as expected, right? So how can I go about differentiating the templates in the CLI? A simple solution I can think of is appending `(community)` to the name of each template I import from the json file.

Reply:
https://github.com/aerogear/graphback/blob/master/packages/create-graphback/src/init/components.ts#L31-L53

Experiment with this methods. Now all will be printed there. 
If we can split them into two categories etc. that will be awesome.